spec/controllers/idv/image_uploads_controller_spec.rb

```diff
@@ -611,6 +707,24 @@
           flow_path: 'standard',
         )
 
+        expect(@irs_attempts_api_tracker).to receive(:track_event).with(
+          :idv_document_upload_submitted,
+          success: false,
+          failure_reason: {
+            general: [I18n.t('doc_auth.errors.alerts.barcode_content_check')],
+            back: [I18n.t('doc_auth.errors.general.fallback_field_level')],
+            hints: true,
```

Comment:
I don't want to make more work for you, but what all this means should probably be documented in the schema doc. It seems like some of these aren't intuitive.

_But_, I wonder if the set of errors is so comprehensive, and will be expanded on in more stories around other proofing events, that there should just be an addendum/appendix outlining what all of the various things mean? In which case it might be sensible to start a follow-up story for this?

Reply:
Yes, I think a follow-up story will be a good approach to adding all failure reasons in the schema doc in addendum/appendix.